src/cow-react/modules/limitOrders/hooks/useTradeFlowContext.ts

```diff
@@ -0,0 +1,69 @@
+import { TradeFlowContext } from 'cow-react/modules/limitOrders/services/tradeFlow'
+import { useWeb3React } from '@web3-react/core'
+import { OrderKind } from '@cowprotocol/contracts'
+import { useLimitOrdersTradeState } from 'cow-react/modules/limitOrders/hooks/useLimitOrdersTradeState'
+import { CurrencyAmount, Percent } from '@uniswap/sdk-core'
+import { Token } from '@uniswap/sdk-core'
+import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useGP2SettlementContract } from 'hooks/useContract'
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from 'state'
+import { useAppData } from 'hooks/useAppData'
+
+// TODO: set proper values
+const deadline = 5 // 5 min
+const allowedSlippage = new Percent(0)
+const recipientAddressOrName = null
+const feeAmountRawValue = 60000000
+
+export function useTradeFlowContext(): TradeFlowContext | null {
+  const { chainId, account, provider } = useWeb3React()
+  const state = useLimitOrdersTradeState()
+  const { allowsOffchainSigning, gnosisSafeInfo } = useWalletInfo()
+  const settlementContract = useGP2SettlementContract()
+  const dispatch = useDispatch<AppDispatch>()
+  const appData = useAppData({ chainId, allowedSlippage })
```

Comment:
Not yet defined/discussed, but we might want to store more info the appData related to limit orders

Reply:
added TODO